docs/querying/sql-functions.md

```diff
@@ -605,11 +605,34 @@ Returns the rank for a row within a window without gaps. For example, if two row
 
 ## DIV
 
-`DIV(x, y)`
+Returns the result of integer division of `x` by `y`.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `DIV(x, y)`
+* **Function type:** Scalar, numeric
 
-Returns the result of integer division of `x` by `y`.
+<details><summary>Example</summary>
+
+  The following calculates integer divisions of `78` by `10`.
+
+  ```sql
+  SELECT DIV(78, 10) as "division"
+  ``` 
+
+  Returns the following:
+
+  | `division` |
+  | -- |
+  | `7` |
+
+</details>
```

Comment:
Comment out the docs (`<!-- -->`) for this function. It is false until Druid 31. We can fix it closer to the release.

Reply:
Keep the admonition.

Reply:
commented out the <Details> box and all its content